packages/app-mobile/services/e2ee/Krypto.react-native.ts

```diff
@@ -0,0 +1,51 @@
+import { KryptoInterface } from '@joplin/lib/services/e2ee/types';
+import crypto from 'react-native-quick-crypto';
+import { HashAlgorithm } from 'react-native-quick-crypto/lib/typescript/keys';
+
+const Krypto: KryptoInterface = {
```

Comment:
- In general we don't "brand" filenames to go around naming conflicts. TypeScript provides ways to deal with this when importing.

- The name should be all lowercase because it's a library of functions (see coding style guide) and should be "crypto.ts"

- It should default-export this constant. Name it cryptoLib if you want, and then `export default cryptoLib`)

Reply:
The `Krypto` was my suggestion, since I didn't know whether ts/js had namespaces. Sorry about that.

Reply:
OK I will rename them.